fivox/eventSource.cpp

```diff
@@ -369,4 +375,132 @@ size_t EventSource::getNumChunks() const
     return _getNumChunks();
 }
 
+bool EventSource::read( const std::string& filename )
+{
+    lunchbox::MemoryMap binaryFile( filename );
+    const size_t size = binaryFile.getSize();
+
+    const size_t nElems = size / sizeof( uint32_t );
+    if( nElems == 0 )
+        LBTHROW( std::runtime_error( filename + " is empty" ));
```

Comment:
This function returnn bool, but some and only some errors throw and exception, this is inconsistent. Since I guess this is a virtual function and you can't change the signature, I think it's better to replace the exceptions with LBWARN and return false.